### DISCUSSION TITLE
Documentation Sharing : definition of the database fields for the different references

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#243] [Show and tell]
all these fields allow to indicate references but do not serve the same purpose (this is my perception)

ISBN : International Standard Book Number - number given by the publishers of the book (international).
 	
ISBN-10 :
	- ISBN-10 is composed of 10 digits.
	- the last digit is obtained by a calculation.
	- an  ISBN-10 can be transformed into an ISBN-13.

ISBN-13 :
	- ISBN-13 begins with 978 or 979.
	- ISBN-13 is composed of 13 digits.
	- the last digit is obtained by a calculation.
	- an ISBN-13 starting with 979 has no equivalent in ISBN-10.

Accession Number : The Accession Number is a number assigned to each work for cataloging purposes, it is 	unique. It is represented in the form of a bar code (local).
	https://en.wikipedia.org/wiki/Accession_number_(cultural_property)

Alternate Id : Allows you to indicate a number that can be used to obtain external information.
	- OpenLibrary - https://en.wikipedia.org/wiki/Open_Library
	- Archival Resource Key (Ark) - https://en.wikipedia.org/wiki/Archival_Resource_Key

Call Number : the Call Number is an address to find the book in the library (local to the library).

Dewey Class Number : Dewey classification system (international).
	https://en.wikipedia.org/wiki/Dewey_Decimal_Classification

LC Control Number :  Library of Congress Classification System (international).
	https://en.wikipedia.org/wiki/Library_of_Congress_Classification

Multi-Volume Set ISBN : internal to BiblioteQ, allows you to indicate the ISBN of a book in several volumes.

**Comment by meteos77** [comment]
Version Française

Tous ces champs permettent d'indiquer des références mais ne servent pas à la même choses. (c'est ma perception)

ISBN : Numéro international normalisé du livre - numéro donné par les éditeurs du livres (internationale).
	https://fr.wikipedia.org/wiki/International_Standard_Book_Number

ISBN-10
	- isbn-10 est composé de 10 chiffres.
	- le dernier chiffre est obtenu par un calcul.
	- un isbn-10 peut être transformé en ISBN-13.

ISBN-13
	- isbn-13 commence par 978 ou 979.
	- isbn-13 est composé de 13 chiffres.
	- le dernier chiffre est obtenu par un calcul.
	- un isbn-13 qui commence par 979 n'a pas d'équivalent en isbn-10.

Numéro d’accès (Accession Number) : Numéro attribué a chaque ouvrage pour le cataloguer, il est unique. Il est représenté sous forme de code à barres. (locale)


Identifiant alternatif (Alternative Id) : Permet d’indiquer un numéro qui peut servir pour obtenir des informations externes.
	- OpenLibrary - https://en.wikipedia.org/wiki/Open_Library (international)
	- Archival Resource Key (Ark) - https://en.wikipedia.org/wiki/Archival_Resource_Key (international)

La Cote (Call Number) : La Cote est une adresse pour trouver le livre dans la bibliothèque. (locale)

Dewey Class Number : Système de classification Dewey (international).
	[https://fr.wikipedia.org/wiki/Classification_d](https://fr.wikipedia.org/wiki/Classification_d%C3%A9cimale_de_Dewey)[écimale_de_Dewey](https://fr.wikipedia.org/wiki/Classification_d%C3%A9cimale_de_Dewey)

LC Control Number :  Library of Congress Classification System (international).
	[https://fr.wikipedia.org/wiki/Classification_de_la_Bibliothèque_du_Congrès](https://fr.wikipedia.org/wiki/Classification_de_la_Biblioth%C3%A8que_du_Congr%C3%A8s)

Multi-Volume Set ISBN : interne à BiblioteQ, permet d'indiquer l'ISBN d'un livre en plusieurs volumes.

**Comment by textbrowser** [comment]
Thanks!
